### DISCUSSION TITLE
is there a way to add submod to the 'Unseen' section?

### DISCUSSION THREAD

**Discussion by Reporry** [Monika-After-Story/MonikaModDev#8916] [general]
I just created a submod yesterday, and I don't know how to add it to the 'Unseen' section :<

init 5 python:
    addEvent(Event(persistent.event_database,eventlabel="monika_rushia",category=["vtuber"],prompt="Uruha Rushia",pool=True,random=True))

label monika_rushia:
   m 3eub  "Hey"
   m 3eub  "Do you know about Rushia?"
   m 2esa  "Well..She is a vtuber."
   m 3hub  "Vtuber of Hololive!"
   m 3hub  "I think she's pretty cute!"
   m 4rtd  "Did you know she has a yandere version?"
   m 2esc  "In that yandere version she's pretty scary"
   m 7eud  "Usually she gets angry because the audience said pettan, cutting board, wall"
   m 7eud "In other words she has yandere mode enabled"
   m 2hksdlb "Anyway, I just found out about Rushia, sorry!"
return
(i wrote like this, is it wrong?)

**Comment by Booplicate** [comment]
You have wrong indentation, that may cause crashes. Also you probably don't want both `random` and `pool` to be set to `True`.

The events are added to the unseen menu automatically when unlocked.
